### DISCUSSION TITLE
How to ignore some fields?

### DISCUSSION THREAD

**Discussion by vdonets-dev** [origin-energy/java-snapshot-testing#176] [Q&A]
Hi guys, could you please suggest how I can ignore some fields?

**Comment by jackmatt2** [comment]
This is something you need to handle yourself and it depends on what libraries you're using.  It could be as simple as a REGEX find/replace before snapshotting the string or something more sophisticated such as [this](https://github.com/origin-energy/java-snapshot-testing?tab=readme-ov-file#example-hibernateserializer) Hibernate example
